content/guides/developer-guides/object-lifecycle-management/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Object Lifecycle Management
+description: "Developer guide for Object Lifecycle Management."
+weight: 40
+disableToc: false
+tags: ["developer guide", "object lifecycle"] 
+---
+## Overview
+This guide is intended for Object Lifecycle feature developers. It provides detailed steps to write new API using OpenSDS. To understand the feature and requirements in detail, please refer to the Object Lifecycle Management [user guide](/guides/user-guides/object-lifecycle-management/) and Object Lifecycle [design spec](https://github.com/opensds/design-specs/blob/master/specs/capri/object_lifecycle_management.md). 
+
```

Comment:
Since this is a developer guide, the pre-requisites for setting up the development environment can be mentioned here. For ex: MongoDB Compass installation and configuration can be mentioned / linked as it is needed for steps ahead.